__tests__/product/product.mutation.test.js

```diff
@@ -387,50 +372,44 @@ describe('Product mutations', () => {
     const productAfterUpdate = updateProduct.data.updateProduct;
     expect(productAfterUpdate).toBeDefined();
     expect(productAfterUpdate).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Bad Baggy' },
-      { __typename: 'Language', value: 'Жахливий Рюкзачечок' },
+      { value: 'Bad Baggy' },
+      { value: 'Жахливий Рюкзачечок' },
     ]);
     expect(productAfterUpdate).toHaveProperty('description', [
-      { value: 'Baggy is so bad', __typename: 'Language' },
-      { value: 'Рюкзачечок - не добрий))', __typename: 'Language' },
+      { value: 'Baggy is so bad' },
+      { value: 'Рюкзачечок - не добрий))' },
     ]);
     expect(productAfterUpdate).toHaveProperty('category', {
-      __typename: 'Category',
       _id: categoryId,
     });
     expect(productAfterUpdate).toHaveProperty('subcategory', {
-      __typename: 'Category',
       _id: subcategoryId,
     });
     expect(productAfterUpdate).toHaveProperty('mainMaterial', [
       {
-        __typename: 'Language',
         value: 'Canvas-400QQ прошита без спеціального матеріалу',
       },
       {
-        __typename: 'Language',
         value: 'Canvas-400QQ padded without a layer water-resistant material',
       },
     ]);
     expect(productAfterUpdate).toHaveProperty('innerMaterial', [
       {
-        __typename: 'Language',
         value: 'Oxford 115',
       },
       {
-        __typename: 'Language',
         value: 'Oxford 115',
       },
     ]);
     expect(productAfterUpdate).toHaveProperty('strapLengthInCm', 90);
     expect(productAfterUpdate).toHaveProperty('pattern', [
-      { value: 'Вишивочка', __typename: 'Language' },
-      { value: 'Embroidery', __typename: 'Language' },
+      { value: 'Вишивочка' },
+      { value: 'Embroidery' },
     ]);
     expect(productAfterUpdate).toHaveProperty('closureColor', 'white');
     expect(productAfterUpdate).toHaveProperty('basePrice', [
-      { value: 777000, __typename: 'CurrencySet' },
-      { value: 7779, __typename: 'CurrencySet' },
+      { value: 777000 },
```

Comment:
Please, don't hardcode values which you assert, instead use properties of the object that you have used for the creation of the product. It will reduce the required changes if some data of the product is going to be changed

Reply:
solved